README.md

```diff
@@ -48,12 +49,14 @@ for probabilistic programs][thesis-doi]. Thesis. University of Cambridge.
 
 Now you can use `stack build`, `stack test` and `stack ghci`.
 
-To use the notebooks in the `notebooks` directory, you will first need `nix`. Then:
+**To view the notebooks, go to the website**. To use the notebooks interactively, you will first need `nix`. Then:
 
 1. Run `nix develop --system x86_64-darwin --extra-experimental-features nix-command --extra-experimental-features flakes`
 
 2. This should open a shell, from which you can run `jupyter-lab` to load the notebooks
 
+Your mileage may vary. 
```

Comment:
It's an English idiom. Roughly means: "I don't guarantee that this will work for you". We can change the wording, but I just wanted to let people know that they might need to do some troubleshooting to get it to work for them.

Reply:
Ah, I see now :) Nice idiom indeed!
On the other hand it would be good to take the setup pain away from a reader's shoulders - in which Nix should help.
I suggest a small reformulation - instead of :


> **To view the notebooks, go to the website**. To use the notebooks interactively, you will first need `nix`. Then:
> 
> 1. Run `nix develop --system x86_64-darwin --extra-experimental-features nix-command --extra-experimental-features flakes`
> 
> 2. This should open a shell, from which you can run `jupyter-lab` to load the notebooks
> 
> Your mileage may vary. 
>

Write:

**To view the notebooks, go to the website**. To use the notebooks interactively:
1. Compile the source: `stack build`
2. If you do not have `nix` [install it](https://nixos.org/download.html).
3. Run `nix develop --system x86_64-darwin --extra-experimental-features nix-command --extra-experimental-features flakes` - this should open a nix shell.
4. Run `jupyter-lab` from the nix shell to load the notebooks

Your mileage may vary. 


It is pretty much the same but keeps the prerequisite to run notebooks in one place.

Reply:
I like this, let's do it.

Reply:
done in https://github.com/tweag/monad-bayes/pull/179/commits/2ef32d9758ce09a48439eed39ef349f7645eb2ac